start-site/src/main/resources/application.yml

```diff
@@ -855,6 +855,35 @@ initializr:
               description: Messaging with JMS
             - rel: reference
               href: https://docs.spring.io/spring-boot/docs/{bootVersion}/reference/htmlsingle/#messaging.jms.artemis
+        - name: Spring for Apache Pulsar
+          id: pulsar
+          compatibilityRange: "[3.0.0,3.1.0-M1)"
```

Comment:
The repo does not currently publish a bom. It instead uses a Gradle [Java Platform](https://docs.gradle.org/current/userguide/java_platform_plugin.html) to handle its versions.